packages/e2e/test/web-extension/extension/background/config.ts

```diff
@@ -9,50 +11,59 @@ import {
   blockfrostTxSubmitProvider,
   blockfrostUtxoProvider
 } from '@cardano-sdk/blockfrost';
-import { Cardano } from '@cardano-sdk/core';
-import { Logger } from 'ts-log';
+import {
+  assetInfoHttpProvider,
+  chainHistoryHttpProvider,
+  networkInfoHttpProvider,
+  rewardsHttpProvider,
+  txSubmitHttpProvider,
+  utxoHttpProvider
+} from '@cardano-sdk/cardano-services-client';
+import { createConnectionObject } from '@cardano-ogmios/client';
 import { createStubStakePoolProvider } from '@cardano-sdk/util-dev';
+import { ogmiosTxSubmitProvider } from '@cardano-sdk/ogmios';
 import axiosFetchAdapter from '@vespaiach/axios-fetch-adapter';
 
-const loggerMethodNames = ['debug', 'error', 'fatal', 'info', 'trace', 'warn'] as (keyof Logger)[];
 const networkIdOptions = [0, 1];
-const stakePoolProviderOptions = ['stub'];
-const networkInfoProviderOptions = ['blockfrost'];
-const txSubmitProviderOptions = ['blockfrost'];
-const assetProviderOptions = ['blockfrost'];
-const utxoProviderOptions = ['blockfrost'];
-const rewardsProviderOptions = ['blockfrost'];
-const keyAgentOptions = ['InMemory'];
-const chainHistoryProviderOptions = ['blockfrost'];
+const stakePoolProviderOptions = ['stub', 'http'];
+const networkInfoProviderOptions = ['blockfrost', 'http'];
+const txSubmitProviderOptions = ['blockfrost', 'http'];
+const assetProviderOptions = ['blockfrost', 'http'];
+const utxoProviderOptions = ['blockfrost', 'http'];
+const rewardsProviderOptions = ['blockfrost', 'http'];
+const chainHistoryProviderOptions = ['blockfrost', 'http'];
 
 const env = envalid.cleanEnv(process.env, {
   ASSET_PROVIDER: envalid.str({ choices: assetProviderOptions }),
+  ASSET_PROVIDER_PARAMS: envalid.json({ default: {} }),
   BLOCKFROST_API_KEY: envalid.str(),
   CHAIN_HISTORY_PROVIDER: envalid.str({ choices: chainHistoryProviderOptions }),
-  KEY_AGENT: envalid.str({ choices: keyAgentOptions }),
-  LOGGER_MIN_SEVERITY: envalid.str({ choices: loggerMethodNames as string[], default: 'info' }),
+  CHAIN_HISTORY_PROVIDER_PARAMS: envalid.json({ default: {} }),
```

Comment:
```suggestion
  CHAIN_HISTORY_PROVIDER_PARAMS: envalid.json<TypeMe!>({ default: {} }),
```

Reply:
I've created a separate ticket ADP-2097 to address this change, as the scope here is just extending the existing pattern.